src/deploy.js

```diff
@@ -79,6 +85,9 @@ module.exports = function(progressData, context, client, storage, config, slackT
       return auth0.validateClients(progress, client, context.clients, config('AUTH0_CLIENT_ID'));
     })
     .then(function() {
+      return auth0.validateConnections(progress, client, context.connections);
```

Comment:
I think you want to run validate and update of the connections before your update the database connections.  That way you can create the database and then update the scripts in one run.

Reply:
Done